packages/ember-application/lib/system/application.js

```diff
@@ -261,42 +263,19 @@ var Application = Namespace.extend(DeferredMixin, {
       this.$ = jQuery;
     }
 
+    // Create subclass of Ember.Router for this Application instance.
+    // This is to ensure that someone reopening `App.Router` does not
+    // tamper with the default `Ember.Router`.
+    this.Router = Router.extend();
```

Comment:
can we comment this as for global mode only, something we can shed in the future?